<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsFileEditor</class>
 <widget class="QWidget" name="PointsFileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">border: 0;</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>758</width>
          <height>405</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">border: 0;
background-color: rgb(43, 43, 43);</string>
           </property>
           <column>
            <property name="text">
             <string>Label</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>X</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Z</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Label:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="labelLineEdit">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="xLineEdit">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="yLineEdit">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="zLineEdit">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPointButton">
         <property name="text">
          <string>Add Point</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>labelLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPointButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPointButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPointButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPointButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
